### DISCUSSION TITLE
installing-docker-desktop-for-windows/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#350] [Announcements]
# installing-docker-desktop-for-windows/

In this post I show how to install Docker Desktop for Windows, including WSL 2, and explore the initial experience of running a container.

https://andrewlock.net/installing-docker-desktop-for-windows/

[a comment by an automated account is not shown]

**Reply by birbilis** [reply]
Maybe meant one could run Docker Desktop for Linux inside a Hyper-V VM (though that would start another VM inside the Linux VM)? Docker Desktop in the past without WSL on Windows would use a VM (was slow compared to WSL2), but not sure if it would use Hyper-V or some other VM engine (e.g. VirtualBox or something)

Anyway, Hyper-V role is not deprecated (nor Hyper-V tech in general), free Hyper-V server was discontinued

[a comment by an automated account is not shown]
